mackerel-plugin-gcp-compute-engine/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"github.com/mackerelio/mackerel-agent-plugins/mackerel-plugn-gcp-compute-engine/lib"
```

Comment:
Import path is wrong (and therefore CI fails). Please fix by `s/mackerel-plugn/mackerel-plugin/`.